src/components/network/network-map.js

```diff
@@ -253,7 +253,9 @@ const NetworkMap = (props) => {
     function renderTooltip() {
         return (
             tooltip &&
-            tooltip.visible && (
+            tooltip.visible &&
+            // Should not display the tooltip for HVDC lines
+            !props.mapEquipments.getHvdcLine(tooltip.equipmentId) && (
```

Comment:
can you define mapEquipments in propTypes please? and if it is not required you can add mapEquipments."?" for security thank you

Reply:
Done